easybuild/easyconfigs/d/DB_File/DB_File-1.831-ictce-5.5.0-Perl-5.18.2.eb

```diff
@@ -11,19 +12,13 @@ toolchain = {'name': 'ictce', 'version': '5.5.0'}
 source_urls = ['http://www.cpan.org/modules/by-module/DB_File/PMQS']
 sources = [SOURCE_TAR_GZ]
 
-dependencies = [('DB', '2.7.7')]
```

Comment:
well, not really...

this dep was not used, since `dependencies` is redefined below

and when I included the 2.7.7 version of DB properly, the tests for `DB_File` were failing

with 4.8.30, they pass...